teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
Hmmm. Well, the thing is with the new version of flake8, the plugin doesn't get invoked automatically. You have to pass `--format=teamcity-messages` to `flake8` to explicitly enable it.